examples/networking/coap/nanocoap_server/Makefile

```diff
@@ -24,10 +24,11 @@ ifeq (,$(filter 1, $(LWIP_IPV4) $(LWIP_IPV6)))
 else
   ifeq (1,$(LWIP_IPV4))
     USEMODULE += ipv4_addr
-
     USEMODULE += lwip_arp
     USEMODULE += lwip_ipv4
-    USEMODULE += netdev_legacy_api
+
+    USEMODULE += netdev_eth
+    USEMODULE += netdev_new_api
```

Comment:
The application should never select this, this is used by a driver to inidcate that it's using the new API